packages/react-scripts/scripts/init.js

```diff
@@ -216,20 +216,24 @@ module.exports = function(
   // Install react and react-dom for backward compatibility with old CRA cli
   // which doesn't install react and react-dom along with react-scripts
   if (!isReactInstalled(appPackage)) {
-    args = args.concat(['react', 'react-dom']);
+    args = args.concat(['react@16.4.2', 'react-dom@16.4.2']);
```

Comment:
Same here

Reply:
This was taken from BRS 7 so that means that version was wrong too

Reply:
Locked to correct prod version